# Práctica Calificada 1

**Tópicos de Finanzas Avanzadas (ECON-421, UPAO 2026-20)**

Miércoles 16 de setiembre de 2026. Cubre las semanas 1 a 3. Puntaje total: 20 puntos. Tiempo sugerido: 45 minutos.

## Reglas del juego

Esta es una evaluación con **libro abierto de verdad**: pueden usar internet, sus apuntes, el repositorio del curso, Python y cualquier IA (ChatGPT, Claude, Gemini, la que quieran). No es trampa, es la forma en que van a trabajar como profesionales.

A cambio, las reglas son dos:

1. **Las respuestas escritas van con sus palabras y aplicadas a su caso.** Una respuesta genérica pegada de una IA, correcta pero que podría ser de cualquier alumno y cualquier empresa, obtiene la mitad del puntaje del ítem. Lo que evalúo es que ustedes entiendan, no que su IA entienda.
2. **El trabajo es individual.** Pueden conversar con una IA, no entre ustedes.

## Cómo se entrega

1. Completa este notebook (respuestas escritas en las celdas de texto, código donde se pida).
2. `Kernel  Restart & Run All` para verificar que corre completo.
3. Exporta a PDF: en Colab, menú Archivo, Imprimir, Guardar como PDF; en VS Code o Jupyter, Ctrl+P e imprimir a PDF.
4. Sube el PDF a Canvas antes del fin de la sesión. Como respaldo, haz también commit del notebook en tu fork.

## Parte 1: los conceptos con tus palabras (4 pts, 1 c/u)

Responde cada pregunta en un máximo de dos oraciones, **con tus palabras y usando tu empresa** (la de tus tareas 2 y 3) como ejemplo donde aplique.

**1.1** La semana 1 dijimos que el precio de mercado y el valor intrínseco de una acción pueden diferir. ¿Por qué esa diferencia no te dice, por sí sola, cuánto puedes ganar comprando la acción?

*Tu respuesta:* Porque esa diferencia también puede incluir un error en mi propia estimación, así que no representa una ganancia segura, además que el mercado puede tardar en acercarse al valor calculado.

**1.2** ¿Por qué la tasa de descuento de tu empresa no es un número que se busca en internet sino un número que se construye? Nombra dos insumos que tuviste que decidir tú al construirla.

*Tu respuesta:* Porque se forma a partir de varios supuestos y estimaciones, no de un solo dato que ya venga dado. Con IFS tuve que definir el ERP de 5.5% y el beta ajustado para calcular el Ke.

**1.3** Explica a un gerente sin formación financiera por qué la utilidad neta de su empresa puede crecer mientras la caja disponible cae.

*Tu respuesta:* Porque una empresa puede registrar ventas aunque todavía no haya cobrado ese dinero y puede gastar más en inversiones haciendo que la caja baje aunque la utilidad suba.

**1.4** ¿Cuál es la diferencia entre lo que tu empresa podría repartir a sus accionistas y lo que efectivamente les reparte? ¿Cómo se llama cada concepto?

*Tu respuesta:* Lo que la empresa podría repartir a sus accionists es FCFE, mientras lo que realmente reparte son dividendos y son distintos porque la empresa no siempre distribuye todo lo que tiene disponible.

## Parte 2: encuentra los errores (6 pts)

Un practicante valoró a la empresa Andina SAC y entregó este trabajo. Sus datos de partida son correctos: EBIT = 50, t = 30 por ciento, depreciación = 10, capex = 15, incremento del capital de trabajo = 5, gasto por intereses = 8, endeudamiento neto = 0, Ke = 12 por ciento, WACC = 9 por ciento.

> **Paso 1.** FCFF = EBIT(1-t) + Dep + Int - Capex - WCInv = 35 + 10 + 8 - 15 - 5 = **33**
>
> **Paso 2.** FCFE = FCFF - Int = 33 - 8 = **25**
>
> **Paso 3.** Como la empresa creció 7 por ciento en los últimos años, uso g = 7 por ciento a perpetuidad. Valor del equity = FCFE por (1+g) entre (WACC - g) = 25 por 1.07 entre 0.02 = **1,337.5**

El trabajo contiene **cuatro errores conceptuales**. Identifícalos (2 pts), explica en una oración por qué cada uno es un error (2 pts), y recalcula el valor del equity corrigiendo todo, con un g perpetuo de 3 por ciento (2 pts). Puedes usar la celda de código.

*Errores y explicaciones:*

1.En el FCFF no se suma el gasto por intereses ya que se calcula antes de repartir la caja entre accionistas y acreedores
2.Para pasar de FCFF a FCFE no se resta el interés completo sino el interés después de impuestos
3.Usar un crecimiento de 7% para siempre no es lo ideal y lo mejor sería usar 3
4.El FCFE se debe descontar al Ke y no al WACC porqya que representa el flujo de los accionistas

In [1]:
# Recalculo corregido (puedes usar utils/finanzas.py si trabajas en tu fork)

# TU CODIGO AQUI
EBIT = 50
t = 0.30
dep = 10
capex = 15
wc_inv = 5
interes = 8
endeudamiento_neto = 0
ke = 0.12
g = 0.03

fcff = EBIT * (1 - t) + dep - capex - wc_inv
fcfe = fcff - interes * (1 - t) + endeudamiento_neto
equity = fcfe * (1 + g) / (ke - g)

print(f"FCFF corregido = {fcff:.2f}")
print(f"FCFE corregido = {fcfe:.2f}")
print(f"Valor del equity = {equity:.2f}")

FCFF corregido = 25.00
FCFE corregido = 19.40
Valor del equity = 222.02


*Valor del equity corregido:* 222.02

## Parte 3: tu empresa (6 pts)

Con la empresa de tus tareas (usa `yfinance` o los resultados que ya tienes de la tarea 2):

**3.1** Calcula el FCFF del último año disponible por la ruta del EBIT. Muestra los insumos que usaste (EBIT, tasa efectiva, depreciación, capex, capital de trabajo). (3 pts)

In [2]:
# TU CODIGO AQUI: FCFF del ultimo anio de tu empresa, ruta EBIT
ebit = 4499.8
t = 0.2177
dep = 450.3
capex = 523.4
wc_inv = 1017.6

fcff = ebit * (1 - t) + dep - capex - wc_inv

print(f"EBIT = {ebit:.1f} MM")
print(f"Tasa efectiva = {t:.1%}")
print(f"Depreciación = {dep:.1f} MM")
print(f"Capex = {capex:.1f} MM")
print(f"Capital de trabajo = {wc_inv:.1f} MM")
print(f"FCFF = {fcff:.1f} MM")

EBIT = 4499.8 MM
Tasa efectiva = 21.8%
Depreciación = 450.3 MM
Capex = 523.4 MM
Capital de trabajo = 1017.6 MM
FCFF = 2429.5 MM


In [3]:
import yfinance as yf

tk = yf.Ticker("IFS")
bal = tk.balance_sheet

deuda = bal.loc["Total Debt"] / 1e6

print("Deuda:")
print(deuda)

fcfe_2025 = 2429.5 - 2064.4 * (1 - 0.2177)

print(f"\nFCFE 2025 sin endeudamiento neto = {fcfe_2025:.1f} MM")

Deuda:
2025-12-31    10995.161
2024-12-31    11818.348
2023-12-31    10875.471
2022-12-31    10474.915
Name: Total Debt, dtype: float64

FCFE 2025 sin endeudamiento neto = 814.5 MM


**3.2** Para valorar el equity de tu empresa, ¿usarías FCFF o FCFE? Justifica en dos o tres oraciones mirando su situación real: cómo ha venido cambiando su deuda y si su FCFE es positivo. (2 pts)

*Tu respuesta:* Usaría FCFF porque la deuda de IFS ha ido cambiando y en 2025 disminuyó de 11,818.3 a 10,995.2 MM y al considerar esa reducción de deuda, el FCFE queda ligeramente negativo resultando más adecuado el FCFF

**3.3** El interés que pagó tu empresa el año pasado, ¿a qué financista fue y por qué NO se resta al calcular el FCFF? (1 pt)

*Tu respuesta:* El interés se pagó a los acreedores de IFS y no se resta en el FCFF porque ese flujo todavía considera el dinero disponible para todos los que financian a la empresa

## Parte 4: tú y tu IA (4 pts)

Esta parte evalúa cómo usaste las herramientas, que es una competencia del curso.

**4.1** Pega el prompt más útil que le diste a una IA durante esta evaluación (o durante las tareas, si hoy no la usaste). (1 pt)

*Prompt:* Estoy calculando el WACC de IFS con datos de Yahoo Finance. La capitalización bursátil me aparece en dólares, pero la deuda y los estados financieros están en soles. ¿Puedo usar esos datos directamente o debo convertirlos a una misma moneda antes de calcular los pesos de deuda y patrimonio? Explícame qué debo corregir y cómo hacerlo.

**4.2** Señala algo que la IA respondió de forma incompleta, genérica o directamente mal, y cómo lo detectaste o corregiste tú. Si crees que todo lo que te dijo estuvo perfecto, explica cómo lo verificaste. (3 pts)

*Tu respuesta:* La IA al inicio tomó la capitalización bursátil de IFS en dólares y la deuda en soles para calcular el WACC y di cuenta al revisar la moneda de cada dato en Yahoo Finance y lo corregí convirtiendo primero la capitalización a soles, para que ambos valores estuvieran en la misma moneda antes de hacer el cálculo.

---

**Antes de entregar:** Restart & Run All, exportar a PDF, subir a Canvas. Commit en el fork como respaldo:

```bash
git add examenes/clase03_pc1.ipynb
git commit -m "PC1"
git push
```